src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/NotificationService.java

```diff
@@ -1907,4 +1911,19 @@ private EmailTemplateNames getIntervenerSendOrderContestedTemplate(IntervenerTyp
             }
         }
     }
+
+    public void sendContestedOrderReadyToReviewToJudge(FinremCaseDetails caseDetails) {
+
+        FinremCaseData caseData = caseDetails.getData();
+        UploadAgreedDraftOrder agreedDraftOrder = caseData.getDraftOrdersWrapper().getUploadAgreedDraftOrder();
+
+        String judgeName = agreedDraftOrder.getJudge();
```

Comment:
looks like `getJudge` returns Judge's email. It is from the `FixedLists-nonprod.json`
```
  {
    "LiveFrom": "01/01/2017",
    "ID": "FR_fl_AssignToJudge",
    "ListElementCode": "juitestjudgedivorce@gmail.com",
    "ListElement": "JUI Test Judge Divorce"
  }
```

```
  "draftOrdersReviewCollection": [
    {
      "id": "9b606284-5681-4d31-bc46-4261f4c583ac",
      "value": {
        "hearingDate": "2024-11-11",
        "hearingTime": "09:00 AM",
        "hearingType": "First Directions Appointment (FDA)",
        "hearingJudge": "juitestjudgefinrem@mailnesia.com",
```

Reply:
@jthmcts how can we lookup the judge name from the FixedList in java?